src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,100 @@
+.contacts {
+  margin: 0 20px 84px;
+
+  @include on-tablet {
+    margin: 0 72px 156px;
+  }
+
+  &-title {
+    text-align: center;
+    font-size: 32px;
+    font-weight: 700;
+    letter-spacing: -1.5px;
+
+    @include on-tablet {
+      font-size: 48px;
+    }
+  }
+
+  &__container {
+    @include on-tablet {
+      flex-direction: row;
+      gap: 25px;
+    }
+
+    @include on-desktop {
+      display: flex;
+      flex-direction: row;
+    }
+
+    &__wrapper {
+      display: flex;
+      flex-direction: column;
+
+      @include on-tablet {
+        flex-direction: row;
+        align-items: flex-start;
+        justify-content: space-between;
+      }
+    }
+  }
+
+  &__container--card {
+    margin-bottom: 48px;
+  }
+
+  &__title {
+    padding-bottom: 16px;
+    font-size: 16px;
+    font-weight: 700;
+
+    &--title {
+      font-weight: 400;
+      margin: 8px 0 4px;
+    }
+  }
+
+  &__wrapper {
+    & > a {
+      text-decoration: none;
+      color: #161616;
+      font-weight: 700;
+      margin-right: 20px;
+
+      &:hover {
+        color: #ae1c23;
+      }
+    }
```

Comment:
Avoid using html tags as selectors, if needed add other classes. [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Don%27t%20use%20tag%20names%20for%20styling)
Also, avoid styling an element depending on its relative position to another [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Styling%20an%20Element%20Depending%20on%20its%20Context)